dali/pipeline/data/tensor_list.h

```diff
@@ -290,6 +291,24 @@ class DLL_PUBLIC TensorList : public Buffer<Backend> {
     tensor_views_.clear();
   }
 
+  /**
+   * @brief Swaps the content of two TensorLists
+   */
+  DLL_PUBLIC inline void Swap(TensorList<Backend> *other) {
```

Comment:
```suggestion
  DLL_PUBLIC inline void Swap(TensorList<Backend> &other) {
```
I think we've abandoned this madness....

Reply:
I wanted to be consistent with `ShareData` signature, and it was not reworked, and I don't want to mess up with this in this PR.

Reply:
This function is gone now anyway.